Add checker for TODO comments without detail/assignee

liamg opened the pull request:
Resolves #434

cristaloleg reviewed (approved):
sgtm

quasilyte commented:
I think it should be marked with an "opinionated" tag, just in case.
It's not always sensible to require this rule to be followed.
We can merge PR and add this tag separately though, as it's marked with experimental anyway.

quasilyte commented on checkers/todoCommentWithoutDetail_checker.go:
```diff
@@ -0,0 +1,52 @@
+}
+
+func (c *todoCommentWithoutCodeChecker) VisitComment(cg *ast.CommentGroup) {
+	comment := strings.TrimSpace(cg.Text())
```
Using `Text()` method is generally slower than using `cg.List[...].Text` fields directly.
I think we can analyze it in line-by-line fashion and avoid TrimSpace call as well since regexp already includes a space skipping part.
What do you think?

liamg replied:
@quasilyte Thanks, I've addressed this. Sorry for the delay. It looks like using the `cg.List[...].Text` fields include the comment initialisers (`//` and `/*`) so I've adjusted the regex to take this into account and everything seems to work.

liamg commented:
@quasilyte Thanks, I've addressed your comment. Sorry for the delay. It looks like the `cg.List[...].Text` fields include the comment initialisers (`//` and `/*`) so I've adjusted the regex to take this into account and everything seems to work.

quasilyte commented:
I have a few more suggestions, but I'm OK with merging it as is, so they can be addressed later.
One of my questions is: this is probably more style-related than correctness-related. So the "diagnostic" tag is quite misleading.

liamg commented:
Cool, I changed it to `style` and added `opinionated` too.
